src/collections/_documentation/error-reporting/configuration/index.md

```diff
@@ -216,15 +216,17 @@ setup that requires proxy authentication.
 {:.config-key}
 ### `http-proxy`
 
+{% unsupported browser %}
 When set a proxy can be configured that should be used for outbound requests.  This is also used for
 HTTPS requests unless a separate `https-proxy` is configured.  Note however that not all SDKs
 support a separate HTTPS proxy.  SDKs will attempt to default to the system-wide configured proxy
 if possible.  For instance, on unix systems, the `http_proxy` environment variable will be picked up.
+{% endunsupported %}
 
 {:.config-key}
 ### `https-proxy`
 
-{% unsupported csharp aspnetcore php %}
+{% unsupported browser csharp aspnetcore php %}
```

Comment:
Given these, you might check w the SDK folks if those SDK's should also be on `http-proxy`, or if those other ones support `http` but not `https`.

Reply:
this note is more for me, but confirmed that csharp and aspnetcore don't need to change, awaiting confirmation about php

Reply:
also confirmed w/ PHP that this doesn't need to change